test/y2storage/filesystems/btrfs_test.rb

```diff
@@ -230,15 +230,33 @@
     end
 
     context "when the filesystem already exists" do
+      let(:result) {}
+
       before do
         allow(filesystem).to receive(:exists_in_probed?) .and_return(true)
       end
 
       it "can not insert a subvolume into an existing hierarchy" do
         filesystem.create_btrfs_subvolume(path2, nocow)
-        filesystem.create_btrfs_subvolume(path3, nocow)
+        result = filesystem.create_btrfs_subvolume(path3, nocow)
+        expect(result).to be_nil
         expect(filesystem.btrfs_subvolumes.map(&:path)).to_not include(path3)
       end
+
+      it "and returns nil" do
+        expect(result).to be_nil
```

Comment:
This test is doing nothing. `#create_btrfs_subvolume` has not been called here at all. I don't think you got how `let` works.